sunpy/instr/aia/read_genx.py

```diff
@@ -0,0 +1,386 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
We decided to pull out all of the junk you need to read these `.genx` files and throw them in a separate file, with the idea being that the way we actually get at this data will probably change in the future. The plan is to only put genx-related stuff in here in order to isolate it from the rest of our code.

So for now, a `read_data` method in our `Response` class will call `read_genx` and when/if this changes, it's just a matter of calling some other function in `read_data`.